### PR TITLE
yaourt: support .pkg.tar extension

### PR DESCRIPTION
**Pull request by lenoch**
pacman's completion already supports this extension so just follow suit.

Motivation: makepkg produces packages (e.g. from AUR) without compression with its PKGEXT='.pkg.tar' setting, which speeds up building and installing them manually.

### CODE REVIEW

File: src/_yaourt [hunk 1]
```diff
@@ -273,7 +273,7 @@ _yaourt() {
     _yaourt_opts_pkgfile=(
         '-d[Skip dependency checks]'
         '-f[Overwrite conflicting files]'
-        '*:package file:_files -g "*.pkg.tar.*(.)"'
+        '*:package file:_files -g "*.pkg.tar*"'
```
Comment (syohex) [inline src/_yaourt]: I suppose it is not necessary to remove `(.)`(matching only files). Why did you remove it ?

Comment (lenoch) [inline src/_yaourt]: Thank you for the explanation, I have just pushed a fixed patch.

**Comment by lenoch** [comment]
I force-pushed a fixed patch to the same branch. Thank you for your review :-)

**Comment by syohex** [comment]
Merged

**Comment by lenoch** [comment]
Thanks!
